### PR TITLE
Admin dashboard with permissions. 

### LINKED ISSUE #92: Admin dashboard
Features: 
- Ability to edit criterias and their categories. 
- Add new admins
- Manual creation of candidate profiles? 
- Should be restricted to users with the "admin" role.

### PR DESCRIPTION
**Pull request by amundlrohne**
Moved to separate page, checks roles, adds criteria patching and feature for updating user role. 

Closes #92.
Partial fix for #82.

### CODE REVIEW

File: apps/api/src/routes/middleware/errorHandler.ts [hunk 1]
```diff
@@ -15,5 +15,5 @@ export function errorHandler(error: Error, request: Request, response: Response,
         return response.status(error.httpStatus).json(error.toResponse());
     }
 
-    return response.status(StatusCodes.INTERNAL_SERVER_ERROR).json('Something went wrong.. Try again later.');
+    return response.status(StatusCodes.INTERNAL_SERVER_ERROR).json('Something went wrong.. Try again later.' + error);
```
Comment (perkynades) [inline apps/api/src/routes/middleware/errorHandler.ts]: I assume the "+ error" is there from debugging, remove it :)

File: apps/frontend/src/containers/userAdministration/UserAdministration.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    return (
+        <>
+            <Panel>
+                <Heading size="large">Brukere og deres roller i innbyggerpanelet</Heading>
+                <div className={style.filter}>
+                    <TextField label="Navn" value={name} onChange={handleName} />
+                    <Select label="Rolle" onChange={handleRole} value={role}>
+                        <option value={EnumUserRole.NAV}>NAV ansatt</option>
+                        <option value={EnumUserRole.Admin}>Administrator</option>
+                    </Select>
```
Comment (perkynades) [inline apps/frontend/src/containers/userAdministration/UserAdministration.tsx:34]: I assume this is going to include regular citizens at one point?

**Review by perkynades — APPROVED** [review]
LGTM

File: apps/api/src/routes/middleware/errorHandler.ts (hunk 1 above)
Comment (amundlrohne) [reply apps/api/src/routes/middleware/errorHandler.ts]: Fixed! :)

File: apps/frontend/src/containers/userAdministration/UserAdministration.tsx (hunk 2 above)
Comment (amundlrohne) [reply apps/frontend/src/containers/userAdministration/UserAdministration.tsx:34]: It definitely could if we were to implement functionality that is relevant to those users. Currently this is only for changing permissions for employees.

**Pull request merged**